### ISSUE TITLE
ABySS 2.1.4 fails to build with GCC 7

### ISSUE THREAD

**Issue by sjackman** [bcgsc/abyss#265]
```
g++ -Wall -Wextra -fopenmp -g -O2  -L/usr/lib/x86_64-linux-gnu/openmpi/lib    -o abyss-bloom-dbg abyss_bloom_dbg-bloom-dbg.o ../DataLayer/libdatalayer.a ../Common/libcommon.a -ldl -lm 

abyss_bloom_dbg-bloom-dbg.o: In function `google::sparse_hashtable<RollingBloomDBGVertex, RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::Identity, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::SetKey, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::find_position(RollingBloomDBGVertex const&) const':
/usr/include/sparsehash/internal/hashtable-common.h:244: undefined reference to `std::tr1::hash<RollingBloomDBGVertex>::operator()(RollingBloomDBGVertex) const'
abyss_bloom_dbg-bloom-dbg.o: In function `google::sparse_hashtable<RollingBloomDBGVertex, RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::Identity, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::SetKey, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::move_from(google::sparse_hashtable<RollingBloomDBGVertex, RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::Identity, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::SetKey, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::MoveDontCopyT, google::sparse_hashtable<RollingBloomDBGVertex, RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::Identity, google::sparse_hash_set<RollingBloomDBGVertex, std::tr1::hash<RollingBloomDBGVertex>, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >::SetKey, std::equal_to<RollingBloomDBGVertex>, google::libc_allocator_with_realloc<RollingBloomDBGVertex> >&, unsigned long)':
/usr/include/sparsehash/internal/hashtable-common.h:244: undefined reference to `std::tr1::hash<RollingBloomDBGVertex>::operator()(RollingBloomDBGVertex) const'
collect2: error: ld returned 1 exit status
```
See https://hub.docker.com/r/bcgsc/abyss/builds/bhv2wgug7cerkaekbnwg2f6/

**Comment by benvvalk** [comment]
fixed in 54d84c4cb406ad446a0e79cbd8d0c336819060bc

will make new release shortly

**Comment by sjackman** [comment]
Thanks, Ben!
